src/sensors.h

```diff
@@ -33,17 +33,31 @@
 struct FuFarmSensorsData
 {
   int32_t light;
+
```

Comment:
I'm really not sure putting comments about the units here is helpful. The units are defined in HomeAssistant.cpp and these could easily get out of sync and be a classic case where the comments are wrong and confuse whoever is looking at the code. Could we change the comments just to refert HomeAssistant.cpp or something similar?

Reply:
This makes sense. I must have missed it. Adding in a followup commit.